extensions/BugModal/template/en/default/bug_modal/attachments.html.tmpl

```diff
@@ -94,12 +95,19 @@
     </tr>
   [% END %]
 </table>
+[% END %]
 
-<div id="attachments-actions">
-  [% IF obsolete_attachments %]
-    <button type="button" id="attachments-obsolete-btn" class="secondary">Show Obsolete Attachments</button>
-  [% END %]
-</div>
-
-[%# BMO - attachment related warnings %]
-[% Hook.process("warnings") %]
+<footer id="attachments-footer">
+  <div id="attachments-actions">
+    [% IF user.id %]
+      <a href="[% basepath FILTER none %]attachment.cgi?bugid=[% bug.id FILTER uri %]&amp;action=enter" id="attachments-add-link">
+        <button type="button" id="attachments-add-btn" class="secondary">Attach New File</button>
```

Comment:
In [bug 1217890](https://bugzilla.mozilla.org/show_bug.cgi?id=1217890) this was changed from a `<button>` to a link to allow for opening in a new tab/window/etc.

It would be better if this were a link styled to look like a button rather than an actual `<button>`.

Reply:
Ah, this one should be okay. It’s a `<button>` wrapped in `<a>` so the link opens in a new tab with Accel+click. For a better a11y, I’ll just add `role="none"` to the `<button>` to make sure it’s just for presentation.

The Vote button has the problem, but I’ll keep it as is because it’s an existing bug and the voting functionality will be removed in the near future.

Reply:
You can find a similar `<a>`-wrapped `<button>` on the User Profile page, btw.

Reply:
Well, the Edit Profile & Preferences button on the User Profile page doesn’t open in a new tab, probably because it’s not `<button type="button">` but `<input type="button">`. I’ll fix that in a separate bug.